fastlane/Fastfile

```diff
@@ -173,28 +173,38 @@ platform :ios do
 
   desc "build Swift API tester"
   lane :build_swift_api_tester do
-      match(
-        type: "development",
-        git_url: ENV["CERTS_REPO_URL"],
-        storage_mode: "git",
-        type: "development",
-        app_identifier: ["com.revenuecat.SwiftAPITesterIOS"],
-        username: ENV["FASTLANE_USER"])
-      sh('xcodebuild', '-workspace', '../APITesters/APITesters.xcworkspace', '-scheme', 'SwiftAPITesterIOS')
-      sh('xcodebuild', '-workspace', '../APITesters/APITesters.xcworkspace', '-scheme', 'SwiftAPITester')
+    match(
+      type: "development",
+      git_url: ENV["CERTS_REPO_URL"],
+      storage_mode: "git",
+      type: "development",
+      app_identifier: ["com.revenuecat.SwiftAPITesterIOS"],
+      username: ENV["FASTLANE_USER"]
+    )
+
+    workspace = '../APITesters/APITesters.xcworkspace'
+    scheme = 'SwiftAPITester'
+
+    sh('xcodebuild', '-workspace', workspace, '-scheme', scheme, '-destination', 'generic/platform=iOS Simulator')
```

Comment:
@aboedo don't fret, I'm waiting for you to merge #953 before any of this.

In this branch, now that the target is cross-platform, we can invoke the builds to different destinations like this.
However this got me thinking: does that actually matter? All these APIs are available on all platforms, so I wonder if we can get away with just one build... and if not, maybe we should build all 4: `iOS`/`tvOS`/`watchOS`/`macOS`. Other than slightly longer builds, there's not really a downside.

Reply:
we have a couple of new APIs for SK2 that are limited to iOS, but we don't have any APIs that aren't available on iOS. 
So doing a single pass in iOS should suffice. If and when we add an API that isn't available on iOS, we can add another line here to build in a different target

Reply:
total nitpick: we could use [fastlane's xcodebuild](https://docs.fastlane.tools/actions/xcodebuild/) to keep it all in fastlane and get input validation for free

Reply:
For context on why we had 2 targets in the first place: 
We originally just had a run script target, because it was the easiest, but that would run only on macOS. 
When Xcode 13.0 came out, our SK2 stuff stopped working on Xcode 13.0, because 13.0 didn't include support for Swift 5.5. 
So I added an iOS target with the same files, very lazily, to get the tests working again, and then when 13.1 came out I re-enabled the other target.

Reply:
Got it. Okay I'll combine these to only run on iOS using fastlane then. The benefit of that is that we shouldn't need `match` for it either.